strat-bot.py

```diff
@@ -33,79 +33,123 @@ def update_if_needed(database):
 async def on_ready():
 	print("We have logged in as {0.user}".format(client))
 
-@client.event
-async def on_message(message):
+#Let this command be run by people with the Testers role only.	
+@client.command()
+@commands.has_role('Testers')
+async def strat(ctx, ,*args):
+	#Not sure why this is global but I kept it because who knows.
 	global stratDatabase
-	messageStr = message.content
-	messageStr = messageStr.lower()
-
-	if message.author == client.user:
+	stratDatabase = update_if_needed(stratDatabase)
+	#I got tired of scrolling. Add your options here
+	team_options_a = ['a','attack','attk','atk','attackers']
+	team_options_d = ['d','defender','def','crim','t']
+	tile_options_c = ['c','cs','cstore','store']
+	tile_options_f = ['factory','f','fac']
+	tile_options_k = ['k','kill','killhouse']
+	teams_merged = team_options_a + team_options_d
+	tiles_merged = tile_options_c + tile_options_f + tile_options_k
+	#No arguments provided. Default to random.
+	if len(args) == 0:
```

Comment:
Instead of this big tree, I think it can be greatly simplified. Something like:
```py
tile = "All"
team = "Both"
for arg in args:
  if arg in teams_options_a:
    team = "Attackers"
  elif arg in team_option_d:
    team = "Defenders"
  elif arg in tile_options_f:
    tile = "Factory"
  ... Etc....
validNum = randomGen(team, tile)
....
```

might also need some extra checks for to many params or duplicates, but those should be easy to do

Reply:
That'd be tons better